### PR TITLE
TTK changes for postman interoperability

### PR DESCRIPTION
**Pull request by ramvr1256**

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -55,27 +55,34 @@ const routes = [
...
-        icon: <FileSearchOutlined />,
-        component: RulesValidation,
+        path: '/apimgmt',
+        name: 'API Management',
+        icon: <ApiOutlined />,
+        component: APIManagement,
         layout: '/admin',
     },
     // {
+    //     path: '/monitoring',
```
Comment (fynmanoj) [inline src/routes.js]: remove commented lines

File: package-lock.json [hunk 2]
```diff
@@ -22,8 +22,9 @@
         "axios": "^1.3.2",
         "bootstrap": "^4.5.3",
         "brace": "^0.11.1",
-        "core-js": "^3.27.2",
```
Comment (fynmanoj) [inline package-lock.json]: remove generated files
if these changes can be automatically generated

File: src/views/outbound/EnvironmentManager.jsx [hunk 3]
```diff
@@ -344,6 +393,34 @@ class EnvironmentManager extends React.Component {
                             }}
                         >
                             <Panel header={this.state.localEnvironments[this.state.selectedEnvironmentIndex] ? this.state.localEnvironments[this.state.selectedEnvironmentIndex].name : 'Choose environment'} key='1'>
+                                <Modal zIndex={1101} title="Export Environment" open={this.state.isImportDialogOpen} onOk={this.handleCloseImportDialog} onCancel={this.handleCloseImportDialog}>
+                                    <Button
+                                        type='default'
+                                        info
+                                        // onClick={e => {
+                                        //     e.preventDefault();
+                                        //     this.environmentFileSelector.click();
```
Comment (fynmanoj) [inline src/views/outbound/EnvironmentManager.jsx]: remove commented lines

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Comment by ramvr1256** [comment]
Requested Changes are done @fynmanoj Kindly review.

**Pull request merged**